addons/website_slides/i18n/website_slides.pot

```diff
@@ -731,8 +731,8 @@ msgstr ""
 #: code:addons/website_slides/static/src/js/tours/slides_tour.js:0
 #, python-format
 msgid ""
-"A good course has structure and a table of content. Your first section will "
-"be the <b>Introduction</b>."
+"A good course has a structure. Pick a name for your first section and click "
```

Comment:
Actually do not remove existing translations, just add new ones. Idea is: people will maybe not update everything, or will not update their JS (no F5), or ... plenty of reasons.

Unnecessary strings will be removed anyway when exporting v15 translations. So just add new ones.

Thanks !

cc @mart-e